### PR TITLE
kills tegustation

### PR DESCRIPTION
**Pull request by retlaw34**
## About The Pull Request

kills the ModularTegustation folder
TODO(not in this pr): remove beefmen

## Why It's Good For The Game

Its pretty messy and its been planned for a while

## Changelog
:cl:
refactor: modulartegustation folder is gone, merged with modularbungalowstation
/:cl:

### CODE REVIEW

**Comment by retlaw34** [comment]
Checks pass, good enough

File: ModularBungalow/clothing/hardsuit.dm [hunk 1]
```diff
@@ -6,8 +6,8 @@
 	icon = 'ModularBungalow/clothing/icons/helmet.dmi'
 	icon_state = "hardsuit0-voidtech"
 	inhand_icon_state = "void_hardsuit"
-	lefthand_file = 'ModularTegustation/Teguicons/kirie_stuff/left.dmi'
-	righthand_file = 'ModularTegustation/Teguicons/kirie_stuff/right.dmi'
+	lefthand_file = 'ModularBungalow/legacy_icons/kirie_stuff/left.dmi'
+	righthand_file = 'ModularBungalow/legacy_icons/kirie_stuff/right.dmi'
```
Comment (Kitsunemitsu) [inline ModularBungalow/clothing/hardsuit.dm:10]: Should have been moved in the Clothes refactor.

**Comment by SqueezyS** [comment]
where were u when tegu died

i was eating dorto

"tegu is kil"
no

**Comment by SqueezyS** [comment]
where were u when tegu died

i was eating dorto

"tegu is kil"
no

**Comment by retlaw34** [comment]
This has so many merge conflicts i dont think its salvagable

**Comment by retlaw34** [comment]
screw this

**Pull request closed without merging**